src/java.desktop/windows/classes/sun/awt/windows/TranslucentWindowPainter.java

```diff
@@ -66,14 +65,10 @@ abstract class TranslucentWindowPainter {
     protected WWindowPeer peer;
 
     // REMIND: we probably would want to remove this later
-    @SuppressWarnings("removal")
     private static final boolean forceOpt  =
-        Boolean.parseBoolean(AccessController.doPrivileged(
-            new GetPropertyAction("sun.java2d.twp.forceopt", "false")));
-    @SuppressWarnings("removal")
+        Boolean.getBoolean(System.getProperty("sun.java2d.twp.forceopt", "false"));
```

Comment:
This file has a now-unused import of `GetPropertyAction`.

Reply:
removed